prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory.component.ts

```diff
@@ -22,6 +22,7 @@ import { CollegeCertification } from '@enrolment/shared/models/college-certifica
 import { CareSetting } from '@enrolment/shared/models/care-setting.model';
 
 import { RegulatoryFormState } from './regulatory-form-state';
+import moment from 'moment';
```

Comment:
Could you please move this import up, see https://github.com/bcgov/moh-prime/pull/992/files/03107320337a138712f353d3db12890afa18bec3#r531743185